components/breadcrumb/stories/breadcrumb.stories.js

```diff
@@ -18,6 +18,33 @@ export default {
 			options: ["compact", "multiline"],
 			control: "inline-radio",
 		},
+    isDragged: {
+      name: "Dragged",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "State",
+      },
+      control: "boolean"
+    },
+		isFocused: {
+      name: "Focused",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "State",
+      },
+      control: "boolean"
+    },
+		isKeyboardFocused: {
+      name: "Keyboard Focused",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "State",
+      },
+      control: "boolean"
+    },
```

Comment:
I didn't want to create separate story instances to show dragged and focus states (@pfulton has mentioned to me that using controls instead of story instances makes it easier for Chromatic to capture everything at one time and saves us on snapshots), but I also didn't want to show the state on the existing stories all the time. So that's why we need the controls, and setting them on an individual item allows us to show them more accurately on only one item, instead of setting focus on all the breadcrumb items at once. Does that make sense?